backend/app/management/commands/syncdb.py

```diff
@@ -92,21 +95,27 @@ def handle(self, *args, **options):
         # THIS IS JUST FOR PROTOTYPING NEVER EVER EVER EVER IN PRODUCTION do this
         superuser = User.objects.create_superuser('admin', password='adminadmin')
 
-        if not values:
+        if not databases:
             print_header('No data found.')
         else:
-            print_header('Importing these values from the spreadsheet')
-
-            header = values[0]
-            values_as_a_dict = [{header[i]: entry for i, entry in enumerate(row)}
-                                for row in values[1:]]
-
-            for row in values_as_a_dict:
-                print(row)
-                photo = Photo(
-                    title=row["title"],
-                    front_src=row["front_src"],
-                    back_src=row["back_src"],
-                    alt=row["alt"],
-                )
-                photo.save()
+            for m, values in enumerate(databases):
+                model_name = spreadsheet_ranges[m]
+                print_header(f'{model_name}: Importing these values from the spreadsheet')
+
+                header = values[0]
+                values_as_a_dict = [{header[i]: entry for i, entry in enumerate(row)}
+                                    for row in values[1:]]
+
+                for row in values_as_a_dict:
+                    print(row)
+                    if model_name == 'Photo' or model_name == 'Photographer':
+                        map_square_name = row.get('map_square_obj', '')
+                        # Returns the object that matches or None if there is no match
+                        row['map_square_obj'] = \
+                            MapSquare.objects.filter(name=map_square_name).first()
+                    if model_name == 'Photo':
+                        photographer_name = row.get('photographer_obj', '')
+                        row['photographer_obj'] = \
+                            Photographer.objects.filter(name=photographer_name).first()
+                    model_instance = eval(f'{model_name}(**row)')
```

Comment:
No `eval`! I'm surprised that's not in our pylint settings; I'll double check. (The basic problem with `eval` is that because you're passing code as a string, it doesn't get linted, typechecked, or checked for syntax before running.)

 You can do this in code by saving the correct model class to a variable instead:
```
if model_name == 'Photo':
    ModelClass = Photo
# ... do other stuff ...
model_instance = ModelClass(**row)
model_instance.save()
```